index.html

```diff
@@ -1,215 +1,245 @@
-                </div>
-            </div>
+      <div class="modal-inner"></div>
+    </div>
 
-            <div class="card port-two">
-                <div class="profile order-1"><img src="image-source/Snapshoot Portfolio (1).png" alt="Snapshoot"></div>
-                <div class="card-port-description order-2">
-                    <h3>Multi-Post Stories</h3>
-                    <div class="profile-description">
-                        <h5>CANOPY</h5>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>Back End Dev</p>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>2015</p>
-                    </div>
-                    <div class="paragraph-description">
-                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    </div>
-                    <ul class="lang">
-                        <li>html</li>
-                        <li class="ruby">Ruby on rails</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button class="btn" type="button">See Project</button>
-                </div>
-            </div>
+    <section class="landing">
+      <header>
+        <nav class="nav-container toolbar-container">
+          <a id="logo" href="#">My Logo</a>
 
-            <div class="card port-three">
-                <div class="profile"><img src="image-source/Snapshoot Portfolio (2).png" alt="Snapshoot"></div>
-                <div class="card-port-description">
-                    <h3>Tonic</h3>
-                    <div class="profile-description">
-                        <h5>CANOPY</h5>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>Back End Dev</p>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>2015</p>
-                    </div>
-                    <div class="paragraph-description">
-                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    </div>
-                    <ul class="lang">
-                        <li>html</li>
-                        <li class="ruby">Ruby on rails</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button class="btn" type="button">See Project</button>
-                </div>
-            </div>
+          <a class="hamburger">
+            <i class="fa fa-solid fa-bars"></i>
+          </a>
+          <ul class="menu-bar">
+            <li class="nav"><a href="#Portfolio">Portfolio</a></li>
+            <li class="nav"><a href="#aboutme">About</a></li>
+            <li class="nav"><a href="#contact-me">Contact</a></li>
+          </ul>
+        </nav>
```

Comment:
- [ ] [OPTIONAL] You have done very great by implementing the hamburger function. But to make it more professional and stick to the Figma design, I suggested that you make it more blurred ☺️ .

Your design 😎 
![blur](https://user-images.githubusercontent.com/99394374/215709524-5bcec73c-8e97-42fb-a3b0-07e22d587b2f.PNG)

Reply:
Thanks for the review @AlphaNtihinduka